include/range/v3/view/transform.hpp

```diff
@@ -166,13 +166,13 @@ namespace ranges
         constexpr /*c++14*/ auto CPP_fun(size)() (
             requires SizedRange<Rng>)
         {
-                return static_cast<range_size_type_t<Rng>>(ranges::size(this->base()));
+                return ranges::size(this->base());
```

Comment:
This looks suspiciously like a greater than 4 space indent to me. :-? Preexisting of course, but since you're changing that line anyway.... (Below, also.)